src/front-end/src/components/MenteeFormAndEmbedder/MenteeFormAndEmbedder.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+import { mockWindowMatchMedia } from '../../helper-methods';
+import MentorFormAndEmbedder from './MenteeFormAndEmbedder';
+
+let container = null;
+
+mockWindowMatchMedia();
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  // cleanup on exiting
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it('has a resend email button', () => {
```

Comment:
This should probably be changed to 
```suggestion
it('has a Generate iFrame Tag button', () => {
```